dev/TeachingTip/TeachingTip.cpp

```diff
@@ -146,6 +146,17 @@ void TeachingTip::OnContentChanged(const winrt::IInspectable& oldContent, const
     }
 }
 
+void TeachingTip::CreateLightDismissIndicatorPopup()
```

Comment:
In general this code needs more comments to help future maintainers. What is this for? What is the strategy/invariant it maintains? Why can't we use the main popup's light dismiss? (I know the answers but only because we talked offline)